data/gh/character/deck/tinkerer.json

```diff
@@ -146,46 +215,130 @@
+              "value": 3,
+              "small": true,
+              "hidden": false
+            },
+            {
+              "type": "custom",
+              "value": "Gain %game.card.experience:1% for each",
+              "small": true,
+              "hidden": false
+            },
+            {
+              "type": "custom",
+              "value": "enemy targeted.",
+              "small": true,
+              "hidden": false
+            },
+            {
+              "type": "element",
+              "value": "dark",
+              "small": true,
+              "hidden": false
+            }
+          ],
+          "hidden": false,
           "enhancementTypes": [
-            "any"
+            "diamond"
           ]
         }
       ],
       "bottomActions": [
         {
-          "type": "custom",
-          "value": "%character.abilities.wip%",
+          "type": "move",
+          "value": 4,
+          "hidden": false,
           "enhancementTypes": [
-            "any"
+            "circle"
           ]
         }
-      ]
+      ],
+      "lost": true
     },
     {
-      "name": "Ilet Shooter",
+      "name": "Net Shooter",
       "cardId": 35,
       "level": 1,
       "initiative": 19,
       "actions": [
         {
-          "type": "custom",
-          "value": "%character.abilities.wip%",
-          "enhancementTypes": [
-            "any"
-          ]
+          "type": "attack",
+          "value": 3,
+          "subActions": [
+            {
+              "type": "area",
+              "value": "(0,1,target)|(1,0,target)|(1,1,target)|(0,0,enhance)",
+              "hidden": false,
+              "enhancementTypes": [
+                "hex"
+              ]
+            },
+            {
+              "type": "range",
+              "value": 3,
+              "small": true,
+              "hidden": false
+            },
+            {
+              "type": "condition",
+              "value": "immobilize",
+              "small": true,
+              "hidden": false
+            },
+            {
+              "type": "custom",
+              "value": "Gain %game.card.experience:1% for each",
+              "small": true,
+              "hidden": false
+            },
+            {
+              "type": "custom",
+              "value": "enemy targeted.",
+              "small": true,
+              "hidden": false
+            }
+          ],
+          "hidden": false
         }
       ],
       "bottomActions": [
         {
-          "type": "custom",
-          "value": "%character.abilities.wip%",
+          "type": "condition",
+          "value": "immobilize",
+          "subActions": [
+            {
+              "type": "specialTarget",
+              "value": "enemyAdjacent",
+              "small": true,
+              "hidden": false
+            }
+          ],
+          "hidden": false,
+          "enhancementTypes": [
+            "diamond"
+          ]
```

Comment:
There seems to be an issue showing the enhancement dot next to negative conditions:
![Screenshot 2025-03-30 at 9 56 03 PM](https://github.com/user-attachments/assets/fa633a7a-9e4b-4700-8d99-ae5c321bdcd7)

Here is the actual card:
![PXL_20250331_025800755 MP](https://github.com/user-attachments/assets/4bb1bfc9-040b-4829-89c6-94173828ab16)

Card 52 is missing it too on the Disarm condition:
![Screenshot 2025-03-30 at 9 54 01 PM](https://github.com/user-attachments/assets/8cf89f15-fe92-492e-9ceb-23818c06d731)

Reply:
will be fixed soon!